special-pages/pages/new-tab/app/omnibar/components/SuggestionsList.js

```diff
@@ -30,12 +30,13 @@ export function SuggestionsList({ id, suggestions, selectedSuggestion, setSelect
                         role="option"
                         id={suggestion.id}
                         class={styles.item}
+                        tabIndex={suggestion === selectedSuggestion ? 0 : -1}
```

Comment:
This fixes the bug you found in Safari/WebKit @shakyShane. I’ll circle back on the tab stop order later as need to have a better think about it. E.g. I’m not really sure what pressing <kbd>Tab</kbd> should do when suggestions list is open.